src/test/java/api/loans/scenarios/CheckInWithdrawnItemTest.java

```diff
@@ -0,0 +1,91 @@
+package api.loans.scenarios;
+
+import static api.support.matchers.ItemMatchers.available;
+import static api.support.matchers.ItemMatchers.awaitingPickup;
+import static api.support.matchers.ItemMatchers.inTransit;
+import static api.support.matchers.ItemMatchers.withdrawn;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static api.support.matchers.LoanMatchers.hasItem;
+import static api.support.matchers.RequestMatchers.openAwaitingPickup;
+import static api.support.matchers.RequestMatchers.openInTransit;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasNoJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.CheckInByBarcodeResponse;
+import api.support.builders.ItemBuilder;
+import api.support.builders.RequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+
+public class CheckInWithdrawnItemTest extends APITests {
+  @Test
+  public void canCheckInAtHomeLocation() {
+    final InventoryItemResource item = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::withdrawn);
+
+    assertThat(item.getJson(), withdrawn());
+
+    loansFixture.checkInByBarcode(item);
+
+    assertThat(itemsClient.getById(item.getId()).getJson(), available());
```

Comment:
Can the status be wrapped in a `is` matcher, to make this read more like a sentence?

```suggestion
    assertThat(itemsClient.getById(item.getId()).getJson(), is(available()));
```

Reply:
I'll add `is` prefix if that's fine.

Reply:
Sure, whichever is easiest for you. Please can it be done for all of the statuses